.github/workflows/ubuntu_build.yml

```diff
@@ -1,4 +1,4 @@
-name: Base Build
+name: Ubuntu build
```

Comment:
revoke changes in file/job name

Reply:
well yes 
Base build doesn't tell the user on the website nothing really 
We will implement Rockyos support and then we will have on the main readme 
Rockyos build
aaaand base build ?

Reply:
therea re other names under base build, not only this, and we also agreed to keed the same naming convection between BCS/MTL/MCM  CI pipelines